### DISCUSSION TITLE
acesso a dados - dapper - dúvida sobre adicionar role a um user

### DISCUSSION THREAD

**Discussion by dhionys-soares** [balta-io/forum#487] [Tira Dúvidas] [answered]
No curso de acesso a dados - dapper, eu tenho dúvidas de como associar um perfil Role a um usuário User, como pedido no desafio.
Seria o caso de realizar um execute com um script insert into na tabela UserRole passando o UserId e o RoleId ?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @dhionyssoares, como vai?
Seria isto mesmo que comentou, quer postar o código que você tentou fazer e te guiamos?
